pysrc/models/metanome_run.py

```diff
@@ -309,7 +309,7 @@ def run_metanome(configuration: MetanomeRunConfiguration, output_fname: str, pip
     elif configuration.algorithm == 'PartialSPIDER':
         algorithm_path = 'PartialSPIDER.jar'
         algorithm_class_name = 'de.metanome.algorithms.spider.SPIDERFile'
-        missing_values = 1000 # TODO: Make this configurable and dependent on the sample size
+        missing_values = 10000 # TODO: Make this configurable and dependent on the sample size
```

Comment:
I feel like we should make a choice if we want to set this to an absurdly high value or choose the value based on something like 50% of the largest column as Prof. Naumann suggested

Reply:
50% of the largest column might be better, but the absurdlyhigh value would have the benefits that we don't have to deal with false negatives

Reply:
Yeah sure, but with the whole mantra of 'Columns are either very different or very similar' I'm not sure if this would make much of a difference.
Then again the ML model has to do the heavy lifting in this case and it is not guaranteed that we don't produce FNs at this stage, as this is a hard problem and I might have overlooked something during feature engineering

Reply:
![grafik](https://user-images.githubusercontent.com/79922493/213865885-3dc1607e-f0af-4fad-83c6-960af27e66c7.png)

yeah I dont really know what might work better on the left is the SCOP Dataset with our previous missing values parameter and on the right is with the increased one.

Reply:
We'll just have to try it out.